### PR TITLE
Feature integrate ap3 docu

### PR DESCRIPTION
**Pull request by bereba**
Some extra descriptions in the data preface for data and parameter-related information from the AP3 documentation. 
Please check if they are consistent regarding your AP1 and AP9 activities.

### CODE REVIEW

**Comment by chrwm** [comment]
Let me know, when this is ready for review.

File: docs/data_preface.md [hunk 1]
```diff
@@ -63,12 +113,35 @@ This *data preface* provides additional information about the syntax of paramete
...
+    Please, link the emission factor column of the processes to this table and the correct column via a foreign key 
+    as explained in the note below `Linking data with foreign keys`. <br>
+
+    Emission factors for ch4 and n2o are to be defined sector- or process-specific as they are more related to attributes such as firing temperature and humidity.
+    Therefore you can directly enter numbers in the respective emission column. If you still want to define ch4 values globally e.g. for similar processes, you can
+    add an extra column to the `global_emission_factors` adding <_ch4> or <_n2o> to the column names. <br>
+
+    If you need to aggregate emission factors from multiple fuels, please add a new column to the table indicating the different
+    fuel emission factors as well as their share in the aggregation based on the assumed potentials.`agg_<percentage>_<factor1>_<percentage>_<factor2>` <br>
+    e.g. agg_30_waste_wood_wood_scraps_industry_70_waste_wood_wood_scraps_commercial_institutional"
```
Comment (chrwm) [inline docs/data_preface.md:134]: This won't work, since column headers are restricted to 52 characters

File: docs/data_preface.md [hunk 2]
```diff
@@ -33,15 +33,65 @@ This *data preface* provides additional information about the syntax of paramete
 
 ## Data
 
+??? note "Information on special parameters"
+
+    === "`conversion_factor_<commodity>`"
+        Having many MIMO processes in the model structure, efficiencies are considered with the parameter `conversion_factor`.
```
Comment (chrwm) [inline docs/data_preface.md:39]:
```suggestion
        Having many Multiple-Input-Multiple-Output (MIMO) processes in the model structure, efficiencies are considered with the parameter `conversion_factor`.
```

File: docs/data_preface.md [hunk 3]
```diff
@@ -33,15 +33,65 @@ This *data preface* provides additional information about the syntax of paramete
...
+        as required by the framework.
+
+    === "`flow_share_min/max/fix`"
+        This parameter can be defined to bound flow shares within the commodity groups of MIMO processes that by default have flexible ratios. <br>
+        e.g. for a hydrogen-ready gas turbine (that can either burn methane or h2 in the MIMO-process) the hydrogen flow for every 
+        timestep could be restricted to a ratio of 0.3 with `flow_share_max=0.3`.
+
+    === "`capacity_p_abs_new_max`"
+
+        Growth rates of processes should be considered with the `capacity_p_abs_new_max` / `capacity_e_abs_new_max` parameter.
```
Comment (chrwm) [inline docs/data_preface.md:56]:
```suggestion
        Growth rates of processes should be considered with `capacity_p_abs_new_max` or `capacity_e_abs_new_max` parameter. 
```

**Review by chrwm — CHANGES_REQUESTED** [review]
Please, check the comments

File: docs/data_preface.md (hunk 1 above)
Comment (bereba) [reply docs/data_preface.md]: Okay true. Is the new description fine for you?

**Review by bereba — COMMENTED** [review]
Your suggestions have been implemented.

**Review by chrwm — APPROVED** [review]
Looks fine now. You can merge

**Pull request merged**